basic/js/join.js

```diff
@@ -0,0 +1,36 @@
+import { USER_DATA } from './userData.js';
+
+const joinEmailInput  = document.querySelector('#email');
+const joinBtn = document.querySelector('#btn_join');
+const $form = document.querySelector('form');
+
+function checkEmailInfo(){
```

Comment:
```suggestion
function checkEmail(){
```

info 역시 별로 도움이 되는 postfix는 아닌 것 같습니다.